CMakeLists.txt

```diff
@@ -36,7 +36,7 @@ endif()
 # ------------------------------------------------------------------------------------------------------
 # "Global" variables
 set(PROJECT_NCI_INCLUDE_PATH "")
-set(PROJECT_NCI_LIB_TARGETS "ncursesw")
```

Comment:
We werw using ncursesw because is ncurses + Unicode (UTF8) support. But for some reason appear strange characters on the background in some computers. It would be re-added in the future.